### PR TITLE
[TS_SELENIUM] Add VNC server to the "eclipse/che-e2e" docker image

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Add VNC server to the "eclipse/che-e2e" docker image

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/14341

### CODE REVIEW

**Review by musienko-maxim — APPROVED** [review]

File: dockerfiles/e2e/entrypoint.sh [hunk 1]
```diff
@@ -12,6 +12,19 @@ if [ -z "$TEST_SUITE" ]; then
...
 
+# Launch display mode and VNC server
+export DISPLAY=':20'
+Xvfb :20 -screen 0 1920x1080x16 > /dev/null 2>&1 &
+x11vnc -passwd 123456 -display :20 -N -forever > /dev/null 2>&1 &
+echo ''
+echo '#######################'
+echo ''
+echo 'For remote debug connect to the VNC server 0.0.0.0:5920'
+echo 'password: 123456'
```
Comment (dmytro-ndp) [inline dockerfiles/e2e/entrypoint.sh]: Can we have the `secret` password - the same as for Java selenium tests, that is for official [NodeChromeDebug image](https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeChromeDebug/Dockerfile#L35)?

File: e2e/README.md [hunk 2]
```diff
@@ -36,9 +36,13 @@
 - export the ```"TS_SELENIUM_BASE_URL"``` variable with "Che" url
 - run command ```"npm run test-docker-mount-e2e"```
 
+## Debug docker launch
+
+The ```'eclipse/che-e2e'``` docker image has VNC server instaled inside. For connect use ```'0.0.0.0:5920'``` adress.
```
Comment (dmytro-ndp) [inline e2e/README.md]: Let's add info about password as well.

Comment (Katka92) [inline e2e/README.md]: `For connecting` or `To connect`.

File: e2e/README.md [hunk 3]
```diff
@@ -36,9 +36,13 @@
...
+
 ## The "Happy Path" scenario launching
 
-**The easiest way to do that is to perform steps which described in the "Docker launch" paragraph. 
+**The easiest way to do that is to perform steps which described in the "Docker launch" paragraph.
```
Comment (Katka92) [inline e2e/README.md]: `... which are described ... `

File: dockerfiles/e2e/entrypoint.sh [hunk 4]
```diff
@@ -12,6 +12,19 @@ if [ -z "$TEST_SUITE" ]; then
     TEST_SUITE=test-happy-path
 fi
 
+# Launch display mode and VNC server
+export DISPLAY=':20'
+Xvfb :20 -screen 0 1920x1080x16 > /dev/null 2>&1 &
+x11vnc -passwd 123456 -display :20 -N -forever > /dev/null 2>&1 &
```
Comment (Katka92) [inline dockerfiles/e2e/entrypoint.sh]: Is it required to set up a password? Wouldn't it be better without it?

**Review by Katka92 — APPROVED** [review]

File: dockerfiles/e2e/entrypoint.sh (hunk 4 above)
Comment (Ohrimenko1988) [reply dockerfiles/e2e/entrypoint.sh]: Thank you, password removed.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**